src/main/java/org/folio/innreach/client/ItemStorageClient.java

```diff
@@ -9,11 +9,14 @@
 
 import org.folio.innreach.client.config.FolioFeignClientConfig;
 import org.folio.innreach.dto.Item;
+import org.springframework.web.bind.annotation.PutMapping;
 
 @FeignClient(name = "item-storage", configuration = FolioFeignClientConfig.class, decode404 = true)
 public interface ItemStorageClient {
 
   @GetMapping("/items/{itemId}")
   Optional<Item> getItemById(@PathVariable("itemId") UUID itemId);
 
+  @PutMapping("/items/{itemId}")
+  void updateItemByItemId(@PathVariable("itemId") UUID itemId, Item item);
```

Comment:
There is ItemService which is capable of updating an item and handling possible "version conflict" exceptions
See CirculationServiceImpl.removeItemTransactionInfo(UUID itemId) method as a sample.